### PR TITLE
add standard documents

### PR DESCRIPTION
**Pull request by F-Shahali**
Adding standard documents
- `CHANGELOG.md`
- `AUTHORS.md`
- `README.md`
- `RELEASE.md`
- `package.json`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-- Item3
-### Changed 
-- Item1
-- Item2
-- Item3
+
+-   `Email` support
+-   `Gmail` support
+-   `Telegram` support
+-   Choose direct or indirect sharing mode
```
Comment (AHReccese) [inline CHANGELOG.md:17]: @F-Shahali 
put them all in the "Added" part.
Every single thing for the first release belongs to the "Added" part.

**Review by AHReccese — CHANGES_REQUESTED** [review]

**Comment by AHReccese** [comment]
@F-Shahali 
Also, fulfil the description of the PR.

File: CHANGELOG.md (hunk 1 above)
Comment (F-Shahali) [reply CHANGELOG.md:17]: Yes, I think I have done exactly the same thing.

**Comment by AHReccese** [comment]
@F-Shahali 
Let me know when you are done with this PR, so I take care of the rest.

**Comment by F-Shahali** [comment]
> @F-Shahali Also, fulfill the description of the PR.

I just mentioned the files added in this PR.
Other documents have been added before.

**Comment by F-Shahali** [comment]
> @F-Shahali Let me know when you are done with this PR, so I take care of the rest.

It's done. You can go ahead.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+| Service name | Support  |
+| ------------ | -------- |
+| Email        | &#x2705; |
+| Gmail        | &#x2705; |
+| Telegram     | &#x2705; |
+
+## Issues & bug reports
+
+Just fill an issue and describe it. We'll check it ASAP! or send an email to [info@openscilab.com](mailto:info@openscilab.com 'info@openscilab.com').
```
Comment (AHReccese) [inline README.md:30]: replace `info@` with `mybutton@`

File: .vscode/settings.json
Comment (AHReccese) [inline .vscode/settings.json:1]: Keep this file locally (add the `.vscode` directory to the `.gitignore` file)
As you know, This folder contains user-specific settings for Visual Studio Code.

**Comment by AHReccese** [comment]
@sepandhaghighi 
Dear Sepand, Since we are about to release v0.1 of the MyButton project, and this PR addresses README.md and other standard documents to have in an open-source project, I would appreciate it if you too could kindly review this PR.

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
-# mybutton
\ No newline at end of file
+## Table of contents
```
Comment (sepandhaghighi) [inline README.md:1]: Remove TOC section from `README.md`

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
-# mybutton
```
Comment (sepandhaghighi) [inline README.md:1]: Add a title section and badge section to `README.md`. Something like this:

```
<div align="center">
    <h1>MyButton</h1>
    <br/>
    <a href="https://discord.gg/xxxx"><img src="https://img.shields.io/discord/xxxxxx.svg" alt="Discord Channel"></a>
    ...
</div>
```

I recommend checking out [Shields.io](https://shields.io/.)

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
Thanks for your efforts 💯 

Take a look at my comments.
Also, don't forget `CNAME` file (Example : https://github.com/openscilab/openscilab.github.io/blob/gh-pages/CNAME)

File: README.md [hunk 5]
```diff
@@ -1 +1,48 @@
...
\ No newline at end of file
+<div align="center">
+
+[![React](https://img.shields.io/badge/react-black?logo=react&style=for-the-badge)](https://reactjs.org/) 
+[![TypeScript](https://img.shields.io/badge/TypeScript-007ACC?logo=typescript&style=for-the-badge)](https://www.typescriptlang.org/)
```
Comment (sepandhaghighi) [inline README.md:4]: I recommend against using a combination of Markdown and HTML for badge links. Please replace these badges with HTML code.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,49 @@
-# mybutton
\ No newline at end of file
+<div align="center">
+<a href="https://reactjs.org/">
+  <img src="https://img.shields.io/badge/react-black?logo=react&style=for-the-badge" alt="React"/>
+</a>
+<a href="https://www.typescriptlang.org/">
+  <img src="https://img.shields.io/badge/TypeScript-007ACC?logo=typescript&style=for-the-badge" alt="TypeScript"/>
```
Comment (sepandhaghighi) [inline README.md:6]: @AHReccese I have a question: Is this the official badge for `TS`? I can't see the `TS` logo very clearly!
I suggest changing the background color of this badge.

Comment (sepandhaghighi) [reply README.md:6]: TypeScript

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (AHReccese) [reply README.md]: Sorry, According to the hell amount of tasks, I forgot to update it previously, I noticed the difference but I forgot to update it. Thank you for your detailed review.

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯 
@AHReccese Please take a final look, if everything is okay `Squash/Merge` it after approval.

**Comment by AHReccese** [comment]
@F-Shahali 
Are you done? if you do, please re-request for review.

**Pull request merged**